docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -1,168 +1,306 @@
 
-```code
-# Check your Markdown syntax
-$ yarn markdown-lint
-# markdown-lint, but also checks that external links work
-$ yarn markdown-lint-external-links
-```
+To navigate to a specific version of the documentation:
 
-## Step 3/4 Create a pull request
+1. View the versions of the documentation listed in the `content` directory by running the following command:
+    
+    ```code
+    $ ls -l content
+    ```
+    
+1. Change to the directory under `content` that contains the version of documentation you want to update.
+    For example, navigate to the directory that corresponds to latest version of Teleport.
 
-Commit your changes and push your branch to `gravitational/teleport` (if you
-work at Teleport) or a fork of `gravitational/teleport`.
+    ```code
+    $ cd content/(=teleport.major_version=).x
+    ```
 
-```code
-# If you are an external contributor, create a fork and push to it.
-# This command assumes that you have created an SSH key pair and stored
-# your public key in GitHub.
-$ git remote add fork ssh://git@github.com/my-user/teleport
-$ git rebase origin/master
-$ git push -u fork my-branch
-```
+1. Switch to the master branch by running the following command:
+    
+    ```code
+    $ git switch master
+    ```
 
-Visit https://github.com/gravitational/teleport and create a pull request (PR)
-against your branch. Our continuous integration (CI) pipeline will automatically
-assign reviewers to your PR. The best way to ensure that your PR is merged is to
-respond to reviewer feedback in a timely manner. If feedback goes unaddressed
-your PR may be closed due to inactivity. Please merge your PR as soon as
-possible once you have received all required approvals.
+    You can use a versioned branch if you want to update documentation for a specific previous version of Teleport.
+    However, it most cases, you should use the `master` branch.
 
-## Step 4/4 Backport your changes
+1. Create a new branch of `gravitational/teleport` for your changes by running a command similar to the following:
 
-Since our docs site is versioned, we want to ensure that any changes you make to
-the most recent version of our documentation are reflected for the versions of
+    ```code
+    $ git switch -c my-branch
+    ```
+
+    You now have a branch for a specific version of the documentation content.
+    Before making any changes, note how the content is organized so you know where to find the files to update.
+    
+    Within the `content/*.*` directory in your branch, there's a `docs` subdirectory. 
+    The `content/*.*/docs` subdirectory contains the files and folders used to produce a specific version of the documentation. 
+    The subdirectory includes the following:
+
+    * The `img/` folder for images used inside the pages.
+    * The `pages/` folder for the Markdown files that contain documentation content. 
```

Comment:
Might be worth mentioning that, under `pages`, is an `includes` directory for partials. These aren't rendered unless they are included

Reply:
👍